tests/generative/test_query_model.py

```diff
@@ -135,7 +136,7 @@ def run_test(query_engines, data, variable, recorder):
     ]
 
     recorder.record_results(
-        len(results), len([r for r in results if r is IGNORE_RESULT])
+        len(results), len([r for (_, r) in results if r is IGNORE_RESULT])
```

Comment:
Oooh. It would be interesting to know how much we can see ignored over a longish run with this fixed.